xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/Translations.xml

```diff
@@ -84,11 +84,11 @@ appWithinMinutes.createApp.parent.hint=The parent of the new application. Leave
 appWithinMinutes.createApp.parent.placeholder=Path.To.Parent
 appWithinMinutes.codeSpace.title=Code
 appWithinMinutes.codeSpace.description=This is the place where the application code is located.
-appWithinMinutes.codeSpace.hiddenPagesInfo=The hidden pages are also listed, even though your current user profile settings prevent it, because otherwise none of the application code pages would be visible as they are all hidden by default.
+appWithinMinutes.codeSpace.hiddenPagesInfo=The hidden pages are also listed, even though your current user profile settings prevents to do so, otherwise none of the application code pages would be visible because they are all hidden by default.
```

Comment:
Grammar problem. The `settings` are the one that `prevent`. It's not the `profile` that `prevents`. (i.e. the plural of `prevent` should be inherited from `settings` and not from `profile`'s singular; drop the `s` in `prevents`)

Reply:
Done.